my-app/src/app/courses-page/courses-list/courses-list.component.css

```diff
@@ -0,0 +1,35 @@
+:host {
+    margin: 0 15%;
```

Comment:
Define those margins as a part of the parent layout. Please make this change to other places as well